Stratis.Bitcoin.Features.RPC.Tests/Controller/BaseRPCControllerTest.cs

```diff
@@ -13,6 +13,9 @@ public IFullNode BuildServicedNode(string dir)
         {
             var nodeSettings = NodeSettings.Default();
             nodeSettings.DataDir = dir;
+            // Best set the data folder again, yes?
```

Comment:
This is odd I don't think it should be needed, double check.

Reply:
Inside of NodeSettings the DataFolder is derived from the DataDir, so only setting the DataDir like this will not update the DataFolder. It's relevant in cases where "NodeSettings.Default" is used.

Reply:
Ah well spotted, but default can take a set of arguments so you can pass the data dir like that.
```
NodeSettings.FromArguments("-datadir=xxx",..)
```

Reply:
ok